third-party/CMakeLists.txt

```diff
@@ -19,24 +19,3 @@
 
 message("Entering ${CMAKE_CURRENT_SOURCE_DIR}")
 
```

Comment:
If that's the only thing left here, we can probably also remove the CMakeLists.txt and the directory.